lzy/long-running/src/main/java/ai/lzy/longrunning/task/OpTaskAwareAction.java

```diff
@@ -0,0 +1,77 @@
+package ai.lzy.longrunning.task;
+
+import ai.lzy.longrunning.OperationRunnerBase;
+import ai.lzy.longrunning.OperationsExecutor;
+import ai.lzy.longrunning.dao.OperationDao;
+import ai.lzy.longrunning.task.dao.OperationTaskDao;
+import ai.lzy.model.db.Storage;
+
+import java.time.Duration;
+import java.util.Map;
+
+import static ai.lzy.model.db.DbHelper.withRetries;
+
+public abstract class OpTaskAwareAction extends OperationRunnerBase {
```

Comment:
Type of action that is connected to a task. All inheritants of this class will be executed by task scheduler.